registers.py

```diff
@@ -281,8 +386,11 @@ def prototype(self):
     def body(self):
         return self.expression
 
-def sympy_to_c(expression):
+def sympy_to_c(expression, sym_to_c = lambda s: "(" + str(s) + ")"):
+    stc = lambda x : sympy_to_c(x, sym_to_c)
     """Implement our own string function, so we can replace 2** with 1<<."""
```

Comment:
```suggestion
    """Implement our own string function, so we can replace 2** with 1<<."""
    stc = lambda x : sympy_to_c(x, sym_to_c)
```
Docstrings should go at the start of the function.

Reply:
Addressed.